ecosystem/sep-0006.md

```diff
@@ -1010,6 +1011,22 @@ Name | Type | Description
 * `too_large` -- deposit/withdrawal size exceeded `max_amount`.
 * `error` -- catch-all for any error not enumerated above.
 
+### Refund Object Schema
+
+Name | Type | Description
+-----|------|------------
+`type` | string | One of `partial` or `full`
+`total_refunded` | string | The total amount refunded to the user, in units of `amount_in_asset`. If `type` is `full`, this amount must match `amount_in`.
+`payments` | array | A list of objects containing information on the individual payments made back to the user. The schema for these objects is defined in the section below.
+
+#### Refund Transactions Object Schema
+
+Name | Type | Description
+-----|------|------------
+`refund_payment_id` | string | The payment ID that can be used to identify the refund payment. This is either a Stellar transaction hash or an off-chain payment identifier, such as a reference number provided to the user when the refund was initiated.
```

Comment:
I think, given that we have an associated `_type` field, that this field could be simply named `id`. This may be less confusing since we're saying less about what the identifier is.